docs/source/asynchronous.rst

```diff
@@ -107,6 +107,23 @@ Python 3 with Tornado
    from tornado.ioloop import IOLoop
    IOLoop().run_sync(f)
 
+If you want to reuse an externally created client you need to run the async
+function using the clients loop which can be done with the ``Client.sync``
+function - e.g.
```

Comment:
Statements like reuse "externally created client" and "using the client's loop" seem odd to me.  For the first I'm not sure why the client is external or why we're reusing it.  I think that this is just a normal client.  In the second case I don't think that we can expect all users to understand what an event loop is. 

Instead, I recommend something like ...

> If you want to use an asyncrhonous function with a synchronous client (one made without the `asynchronous=True` keyword) then you should use the `Client.sync` function. 

I've also been warned off of using `e.g.` in docs.  Apparently it's not universally understood by many people for whom English is not their first language. 

I think that the example is great.